lib/config-schema.ts

```diff
@@ -149,6 +149,7 @@ const ConfigSchema = z
           powercycleCentral: z
             .object({
               active: z.boolean(),
+              schedules: z.record(z.string(), z.string()).default({}),
```

Comment:
Shouldn't this be within the `config` block? It's also dependent on the parser selection. `strict` is just the only parser implementation atm.

If you want schedules shareable between configs, I'd be more inclined to introduce an account level setting (perhaps even as a generic pre-processing thing so you could define any field with a variable)

Reply:
Moving to under config. KISS on the reuse, for now.